bundle/src/main/java/com/adobe/acs/commons/errorpagehandler/impl/ErrorPageHandlerImpl.java

```diff
@@ -306,12 +306,18 @@ public String findErrorPage(SlingHttpServletRequest request, Resource errorResou
      */
 	private String findErrorsPath(SlingHttpServletRequest request, Resource errorResource) {
 		final String errorResourcePath = errorResource.getPath();
-		final Resource page = findFirstRealParentOrSelf(request, errorResource);
+		Resource real = findFirstRealParentOrSelf(request, errorResource);
 
         String errorsPath = null;
-        if(page != null) {
-            log.trace("Found page is [ {} ]", page.getPath());
-	        final InheritanceValueMap pageProperties = new HierarchyNodeInheritanceValueMap(page);
+        if (real != null) {
+            log.trace("Found real resource at [ {} ]", real.getPath());
+            if (!JcrConstants.JCR_CONTENT.equals(real.getName())) {
```

Comment:
If I see it right this will always be true because `findFirstRealParentOrSelf` will always return the plain page path. Shouldn't this logic be added to the `findFirstRealParentOrSelf` method instead?